DietPi-Software | Add new software title: Jellyfin (ID=178)

MichaIng opened the pull request:
**Status**: Ready for user feedback

**ToDo**:
- [x] Online docs: https://dietpi.com/phpbb/viewtopic.php?p=26255#p26255
- [x] Wiki list entry: https://github.com/MichaIng/DietPi/wiki/DietPi-Software-list

**Commit list/description**:
+ DietPi-Software | Add new software title: Jellyfin (ID=178)
+ DietPi-Software | Minor coding enhancements and further G_EXEC implementation for error handling and transparency/verbosity reasons
+ DietPi-Services | Add jellyfin service control

MichaIng committed:
v6.32

+ DietPi-Software | Add new software title: Jellyfin (ID=178)
+ DietPi-Software | Minor coding enhancements and further G_EXEC implementation for error handling and transparency/verbosity reasons

Thielak commented:
Just as a heads up Jellyfin's just released the 10.6.0 update, which brings major changes to how many of its components work.
It would probably be a good idea to wait a bit while they iron out some of the inevitable issues that come with such a massive update.

MichaIng committed:
v6.32

+ DietPi-Software | Jellyfin: GPG key is required in binary format
+ DietPi-Software | Jellyfin: Remove GPG key and APT repo on uninstall

MichaIng commented:
Many thanks for your hint. However I expect releases to the stable/main component to be stable, while there is a "unstable" component which should cover sufficient testing. Also there are no older binaries stored in the repo, hence it would require a different download and install method temporarily, not something I'm gonna da 😉.

Also note that DietPi v6.32 release will take some weeks, probably enough for another bugfix release of Jellyfin 😄.

MichaIng committed:
v6.32

+ DietPi-Services | Add jellyfin service control

MichaIng committed:
v6.32

+ DietPi-Software | Jellyfin: Disable file logging (WIP)

MichaIng commented:
Aiming to disable file logging since logging to systemd-journald is done anyway, viewable via `journalctl -u jellyfin`. However not sure how to achieve with these Serilog configs: https://forum.jellyfin.org/t/how-to-disable-file-logging/2556
**EDIT: Not cleanly possible currently.**

MichaIng committed:
v6.32

+ DietPi-Software | Jellyfin: No chance to disable file logging cleanly currently, lets leave it enabled

MichaIng committed:
v6.32

+ DietPi-Software | Jellyfin: Add user to "dietpi" group and set home dir to new data dir
+ DietPi-Software | Jellyfin: Download test media and enhance the functions code a bid

MichaIng committed:
v6.32

+ CHANGELOG | Jellyfin: A FOSS web interface media streaming server available for install

MichaIng committed:
v6.32

+ DietPi-Software | Jellyfin: Remove leftover symlink on uninstall

MichaIng committed:
v6.32

+ DietPi-Software | Jellyfin: Add online docs URL
